granary/bluesky.py

```diff
@@ -161,25 +164,28 @@ def from_as1(obj, from_url=None):
         banner = url
         break
 
-    url = util.get_url(obj) or obj.get('id') or ''
+    url = util.get_url(obj)
+    id = obj.get('id')
+    if not url and id:
+      url = ''
+      parsed = util.parse_tag_uri(id)
+      if parsed:
+        url = f'http://{parsed[0]}'
```

Comment:
honestly not sure if we could get away with https here, or if we need http. trying to remember/imagine which domains we'll see tag URIs on here, if any. I guess if I don't know for sure, we should default to http?

Reply:
I put this in without really thinking about it but I'm actually leaning towards it being https - I don't like relying on endpoints to upgrade connections and the [HTTP part of the spec](https://atproto.com/specs/xrpc#security-and-privacy-considerations) does say to always use HTTPS, though it makes no mention as to whether this would ever be enforced.

All that being said, the protocol prefix here is literally never used, it's just to keep `url_to_did_web` happy! So I think I'll just change it to `https` now and we can revisit if needed?

Reply:
> All that being said, the protocol prefix here is literally never used, it's just to keep `url_to_did_web` happy! So I think I'll just change it to `https` now and we can revisit if needed?

Lol fair. Maybe comment that? Then sgtm!